entities/sendlogs.go

```diff
@@ -0,0 +1,19 @@
+package entities
+
+import "time"
+
+type SendLogs struct {
```

Comment:
I think we should be consistent with the names and remove the s at the end, also the file name.